config/server.ts

```diff
@@ -15,6 +15,11 @@ export function runtimeAssert(
 export const DATABASE_URL: string = process.env.DATABASE_URL || "";
 runtimeAssert(DATABASE_URL, "DATABASE_URL is required");
 
+export const REDIS_URL: string = process.env.REDIS_URL || "";
+export const REDIS_USERNAME = process.env.REDIS_USERNAME;
+export const REDIS_PASSWORD = process.env.REDIS_PASSWORD;
+runtimeAssert(REDIS_URL, "REDIS_URL is required");
```

Comment:
```suggestion
runtimeAssert(REDIS_URL, "REDIS_URL is required");

export const REDIS_USERNAME = process.env.REDIS_USERNAME || undefined;
export const REDIS_PASSWORD = process.env.REDIS_PASSWORD || undefined;
```
Make the defaults clear